ui/README.md

```diff
@@ -0,0 +1,57 @@
+# ui
```

Comment:
The documentation should be about how to run the UI plus how to configure it for the development.